src/translib/transformer/sw_vlan.go

```diff
@@ -810,14 +810,34 @@ func intfVlanMemberRemoval(swVlanConfig *swVlanMemberPort_t,
 
                 case reflect.TypeOf(ocbinds.OpenconfigInterfaces_Interfaces_Interface_Ethernet_SwitchedVlan_Config_TrunkVlans_Union_String{}):
                     val := (trunkVlanUnion).(*ocbinds.OpenconfigInterfaces_Interfaces_Interface_Ethernet_SwitchedVlan_Config_TrunkVlans_Union_String)
-                    vlanName := "Vlan" + val.String
-                    err = validateVlanExists(inParams.d, &vlanName)
-                    if err != nil {
-                        errStr := "Invalid VLAN: " + val.String
-                        err = tlerr.InvalidArgsError{Format: errStr}
-                        return err
+                    vlansList := strings.Split(val.String, ",")
```

Comment:
Please use a function for the common code.

Reply:
done